code/datums/antagonists/antag_datum.dm

```diff
@@ -8,16 +8,29 @@
 	var/can_coexist_with_others = TRUE			//Whether or not the person will be able to have more than one datum
 	var/list/typecache_datum_blacklist = list()	//List of datums this type can't coexist with
 
-
 /datum/antagonist/New(datum/mind/new_owner)
-	. = ..()
 	typecache_datum_blacklist = typecacheof(typecache_datum_blacklist)
 	if(new_owner)
 		owner = new_owner
 
+/datum/antagonist/Destroy()
+	if(owner)
+		LAZYREMOVE(owner.antag_datums, src)
+	owner = null
+	return ..()
+
+/datum/antagonist/proc/can_be_owned(datum/mind/new_owner)
```

Comment:
NOT ANY MORE THERE ISN'T
besides, using a mind proc for something that will change drastically based on what the datum itself expects is silly, because you can't special-case it for every antag datum.